pkg/costmodel/router.go

```diff
@@ -1599,6 +1599,7 @@ func Initialize(additionalConfigWatchers ...*watcher.ConfigMapWatcher) *Accesses
 	a.Router.GET("/costDataModelRange", a.CostDataModelRange)
 	a.Router.GET("/aggregatedCostModel", a.AggregateCostModelHandler)
 	a.Router.GET("/allocation/compute", a.ComputeAllocationHandler)
+	a.Router.GET("/allocation/compute/summary", a.ComputeAllocationHandlerSummary)
```

Comment:
Does registering this endpoint here cause any problems with the closed source build?

Reply:
Nope! The closed source is /allocation/summary. If you registered _that_ endpoint here it would fail, but /allocation/summary should be elsewhere registered.